scripts/make-guac.sh

```diff
@@ -430,18 +539,62 @@ docker run --name guacd \
     -v "${GUAC_DRIVE}":"${GUAC_DRIVE}" \
     -d "${DOCKER_GUACD_IMAGE}" | log
 
-
 # Starting guacamole container
+
 log "Starting guacamole container, ${DOCKER_GUACAMOLE_IMAGE}"
-docker run --name guacamole \
-    --restart unless-stopped \
-    --link guacd:guacd \
-    -v "${GUAC_HOME}":/guac-home \
-    -e GUACAMOLE_HOME=/guac-home \
-    -e LDAP_HOSTNAME="${LDAP_HOSTNAME}" \
-    -e LDAP_PORT="${LDAP_PORT}" \
-    -e LDAP_USER_BASE_DN="${LDAP_USER_BASE},${LDAP_DOMAIN_DN}" \
-    -e LDAP_USERNAME_ATTRIBUTE="${LDAP_USER_ATTRIBUTE}" \
-    -e LDAP_CONFIG_BASE_DN="${LDAP_CONFIG_BASE},${LDAP_DOMAIN_DN}" \
-    -e LDAP_GROUP_BASE_DN="${LDAP_GROUP_BASE},${LDAP_DOMAIN_DN}" \
-    -d -p 8080:8080 "${DOCKER_GUACAMOLE_IMAGE}" | log
+
+if [[ -n $LDAP_HOSTNAME ]]
+then
+    log "Using LDAP authentication, ${LDAP_HOSTNAME}"
+    docker run --name guacamole \
+        "${params_begin[@]}" \
+        -e LDAP_HOSTNAME="${LDAP_HOSTNAME}" \
+        -e LDAP_PORT="${LDAP_PORT}" \
+        -e LDAP_USER_BASE_DN="${LDAP_USER_BASE},${LDAP_DOMAIN_DN}" \
+        -e LDAP_USERNAME_ATTRIBUTE="${LDAP_USER_ATTRIBUTE}" \
+        -e LDAP_CONFIG_BASE_DN="${LDAP_CONFIG_BASE},${LDAP_DOMAIN_DN}" \
+        -e LDAP_GROUP_BASE_DN="${LDAP_GROUP_BASE},${LDAP_DOMAIN_DN}" \
+        "${params_mysql[@]}" \
+        "${params_end[@]}" | log
+elif [[ -n $SAML_IDP_METADATA_URL ]]
+then
+    log "Using SAML authentication, ${SAML_IDP_METADATA_URL}"
+
+    # Add CATALINA_CONF with x-forwarded-proto configuration docker container
+    log "Linking updated server.xml file with x-forward-proto configuration"
+    params_begin+=(-v "${CATALINA_CONF}":"${CATALINA_CONF}")
```

Comment:
this would need to change accordingly also...

```suggestion
    params_begin+=(-v "/etc/cfn/files/tomcat/conf/server.xml:${CATALINA_CONF}")
```

Reply:
Okay.  I'll update.